### PR TITLE
Fix conan tools.replace_in_file() patch

### PR DESCRIPTION
**Pull request by tt4g**
I noticed that `tools.replace_in_file()` failed when trying to build the master branch with conan.
Patch must be updated.

### CODE REVIEW

**Comment by tt4g** [comment]
Note that merging this branch will cause version 3.4.0 of the source ‘tools.replace_in_file()‘ to fail.

File: conanfile.py [hunk 1]
```diff
@@ -43,11 +43,12 @@ def source(self):
         # This small hack might be useful to guarantee proper /MT /MD linkage
         # in MSVC if the packaged project doesn't have variables to set it
         # properly
+        cmake_project = "project(Fruit VERSION %s LANGUAGES CXX)" % (self.version,)
         tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
```
Comment (poletti-marco) [inline conanfile.py]: Instead of using replace_in_file, can we add another if to the CMakeLists.txt file, as in the other PR?

Comment (tt4g) [reply conanfile.py]: Fixed in my last commit: 6d8bcbdd77352656446a496b4a4f86e459dbb0db

However, probably conflict with my other PR.

Comment (poletti-marco) [reply conanfile.py]: Thanks!

**Pull request merged**